test/system/archive_baseline.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/sh -f
+
+echo
+
+if [ $# -ne 1 ]; then
+    echo "Invoke archive_baseline.sh -help for usage."
+    exit 1
+fi
+
+if [ $1 == "-help" ] || [ $1 == "--help" ]; then
+cat << EOF1
+NAME
+
+	archive_baseline.sh - archive pretag baselines to set locations on
+                              hobart, izumi and derecho.
+
+
+SYNOPSIS
+
+	archive_baseline.sh TAGNAME
+	  [-help]
+
+
+ENVIROMENT VARIABLES
+
+	CESM_TESTDIR - Directory that contains the CESM finished results you wish to archive.
+	CAM_FC      - Compiler used, only used on hobart and izumi (PGI,NAG), where the compiler
+                      name is appended to the archive directory.
+
+
+BASELINE ARCHIVED LOCATION
+
+	hobart, izumi:     /fs/cgd/csm/models/atm/cam/pretag_bl/TAGNAME_pgi
```

Comment:
i know this is just copied over from cam, but maybe this is a good opportunity to replace references to "pgi" with "gnu" in this script?

Reply:
Replaced that and deleted the obsolete machine "hobart"